guides/common/modules/snip_step-parameter-ak.adoc

```diff
@@ -0,0 +1,3 @@
+. Click the *Parameters* tab.
+Add the `kt_activation_keys` parameter and enter the name of the activation key as the value.
```

Comment:
I know that Katello is now suggesting available keys under the current organization, but would it be helpful to mention that the keys must be under the same organization as the host is or is going to be?

Reply:
Yes, that's a good point because I don't think that Foreman is aware that the parameter value is an AK when it's entered as a parameter.